lib/generators/logidze/install/templates/migration.rb.erb

```diff
@@ -16,17 +16,26 @@ class <%= @migration_class_name %> < ActiveRecord::Migration
     end
 
     execute <<-SQL
-      CREATE OR REPLACE FUNCTION logidze_version(v bigint, data jsonb) RETURNS jsonb AS $body$
+      CREATE OR REPLACE FUNCTION logidze_version(v bigint, data jsonb, blacklist text DEFAULT '', whitelist text DEFAULT '') RETURNS jsonb AS $body$
```

Comment:
Yes, i used blacklist text[] , but got problems with how to transform ruby string to pl pgSQL array and decided to use strings